src/Header/HeaderItem.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react'
+import style from './Header.module.css'
+import classNames from 'classnames'
+
+const HeaderItem = ({ title, submenu }) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const onArrowClickHandler = () => {
+    setIsOpen(!isOpen)
+  }
+  const item = submenu.map(item => {
+    if (item === 'View all Projects') {
```

Comment:
This is an awful way to do it. Do not rely on data that can change at any moment.